src/classes/query.py

```diff
@@ -257,6 +302,48 @@ def relative_path(self):
         # Convert path to relative (based on current working directory of Python)
         return os.path.relpath(file_path, info.CWD)
 
+    def getAudioData(self):
+        # Ensure that UI attribute exists
+        if "ui" not in self.data:
+            self.data["ui"] = {}
+            self.save()
+
+        audio_data = self.data["ui"].get("audio_data", False)
```

Comment:
`audio_data = self.data["ui"].get("audio_data", [])` ??? to go ahead and init?

Reply:
If I init `audio_data` to [-999], I'll have to empty the list again to store data. I think [-999] should just be saved to the file.